snmp_juniper/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: snmp_juniper
+
+## Overview
+
+Collect SNMP metrics from Juniper appliances, including:
+
+- Juniper EX Ethernet Switches
+- Juniper MX Routers
+- Juniper SRX Firewalls
+
+For details of monitored metrics see the [SNMP integration tile][2]
+
+## Setup
+
+To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.
+
+## Data Collected
+
+### Metrics
+
+For details of monitored metrics see the [SNMP integration tile][2]
+
+### Service Checks
+
+Snmp Juniper does not include any service checks.
+
+### Events
+
+Snmp Juniper does not include any events.
```

Comment:
nit: remove SNMP and just say `Juniper does not include...` 

I think the SNMP piece here is really just an implementation detail to us, I don't want customers thinking about the Juniper tile as SNMP Juniper.